index.html

```diff
@@ -199,5 +191,40 @@ <h1 class="about-title">About me</h1>
                 </div>
         </div>
     </section>
+    <section class="contact">
+        <div class="cont">
+            <h3 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
- [ ] Please fix your design to match the original one as requested by the previous reviewer, your actual text here is not well styled at all, look at the image below, and consider using the same alignment as used in the Figma design
- [ ] the second is your background color, it seems to be a bit gray which is not the case with the Figma design background.

<img width="818" alt="p1" src="https://user-images.githubusercontent.com/77843050/205961650-d4e2fbb1-174f-4c6e-80fa-cc0a6f2fbb87.png">